networkx/algorithms/coloring/equitable_coloring.py

```diff
@@ -384,8 +384,8 @@ def procedure_P(V_minus, V_plus, N, H, F, C, L, excluded_colors=None):
 
 
 def equitable_color(G, num_colors):
-    """Provides equitable (r + 1)-coloring for nodes of G in O(r * n^2) time
-    if deg(G) <= r. The algorithm is described in [1]_.
+    """Provides equitable (r + 1)-coloring for nodes of `G` in O(r * n^2) time
```

Comment:
Should this be a short one-line description followed by a blank line and then this sentence?
And I guess it should say what `r` is (it's not an input).

Reply:
This is a very good suggestion, thanks! I've reorganized the info a bit, including condensing the first line to a single sentence and explicitly defining `r` (though please double check my terminology - is it okay to "maximum degree of a graph"?)